<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planejamento Experimental</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>box_ico.ico</normaloff>box_ico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tabela_planejamento">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>341</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salvar planejamento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_salvar_planejamento">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
